tracing-subscriber/src/reload.rs

```diff
@@ -125,12 +125,8 @@ where
     }
 }
 
-impl<L, S> Layer<L, S>
-where
-    L: crate::Subscriber<S> + 'static,
-    S: Collector,
-{
-    /// Wraps the given `Subscriber`, returning a `Subscriber` and a `Handle` that allows
+impl<L> Layer<L> {
```

Comment:
type name probably ought to change to `Subscriber` (and maybe the type parameter from `L` to `S` as well?)